contracts/interfaces/handlers/IBosonAccountHandler.sol

```diff
@@ -97,17 +97,20 @@ interface IBosonAccountHandler is IBosonAccountEvents {
     function createAgent(BosonTypes.Agent memory _agent) external;
 
     /**
-     * @notice Updates a seller, with the exception of the active flag.
-     *         All other fields should be filled, even those staying the same.
+     * @notice Updates treasury address, if changed. Puts admin, operator, clerk and AuthToken in pending state, if changed.
```

Comment:
Since the protocol has several formal state machines, I would use the word “queue” rather than “state” to avoid any confusion.

Reply:
renamed